core/src/test/scala/org/apache/spark/sql/delta/DeltaParquetFileFormatSuite.scala

```diff
@@ -74,7 +76,8 @@ class DeltaParquetFileFormatSuite extends QueryTest
 
         val fs = addFilePath.getFileSystem(hadoopConf)
         val broadcastDvMap = spark.sparkContext.broadcast(
-          Map(fs.getFileStatus(addFilePath).getPath().toUri -> dv)
+          Map(fs.getFileStatus(addFilePath).getPath().toUri ->
+            DeletionVectorDescriptorWithFilterType(dv, RowIndexFilterType.IF_CONTAINED))
```

Comment:
Can we extend this to test to test IF_NOT_CONTAINED filter as well? May be one more test combination.

Reply:
Done!